lib/parser/source/map/definition.rb

```diff
@@ -9,13 +9,13 @@ class Map::Definition < Map
       attr_reader :name
       attr_reader :end
 
-      def initialize(keyword_l, operator_l, name_l, end_l)
+      def initialize(keyword_l, operator_l, name_l, body_l, end_l)
```

Comment:
not sure if it's worth making this class so generic. Maybe let's create a new class? This way you could give it a very specific attribute for `=` character location.